fastlane/lib/fastlane/actions/synchronize_provisioning.rb

```diff
@@ -4,7 +4,7 @@ module SharedValues
       MATCH_PROVISIONING_PROFILE_MAPPING = :MATCH_PROVISIONING_PROFILE_MAPPING
     end
 
-    class MatchAction < Action
+    class SynchronizeProvisioningAction < Action
```

Comment:
Wondering if `SynchronizeCodeSigning` is more appropriate?

Reply:
@getaaron also had a great suggestion - using the word `sync` rather than `synchronize` because it is easier to spell. Thoughts?

Reply:
`sync` sounds good